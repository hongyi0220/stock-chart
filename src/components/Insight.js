import React from 'react';
import {
  Transition,
  Icon,
  Card,
  Image,
} from 'semantic-ui-react';

const Insight = () => {
  return (
    <div className="stock-insight-card-wrapper">
      <Transition animation='fade down' duration={600} transitionOnMount={true}>
        <Card>
          <Image src='./img/company-logo-placeholder.jpg'/>
          <Card.Content>
            <Card.Header>ABC</Card.Header>
            <Card.Meta>
              <span className='date'>IPO date: 2015</span>
            </Card.Meta>
            <Card.Description>
              Week of 2018-03-29: <br />
              open: 90.6100<br />
              high: 95.1390<br />
              low: 88.4000<br />
              close: 91.2700<br />
              adjusted close: 90.8771<br />
              volume: 207104405<br />
              dividend amount: 0.0000
            </Card.Description>
          </Card.Content>
          <Card.Content extra>
            <a>
              <Icon name='chart line' />
              Bull
            </a>
          </Card.Content>
        </Card>
      </Transition>
    </div>
  );
};

export default Insight;
