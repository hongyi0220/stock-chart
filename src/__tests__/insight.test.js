/* eslint-disable */
import React from 'react';
import Insight from '../components/Insight';

test('Insight should render', () => {
  const wrapper = shallow(<Insight />);
  expect(wrapper).toMatchSnapshot();
});
