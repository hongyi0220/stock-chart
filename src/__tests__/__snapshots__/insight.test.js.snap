// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Insight should render 1`] = `
<div
  className="stock-insight-card-wrapper"
>
  <Transition
    animation="fade down"
    duration={600}
    mountOnShow={true}
    transitionOnMount={true}
    unmountOnHide={false}
    visible={true}
  >
    <Card>
      <Image
        as="img"
        src="./img/company-logo-placeholder.jpg"
        ui={true}
      />
      <CardContent>
        <CardHeader>
          ABC
        </CardHeader>
        <CardMeta>
          <span
            className="date"
          >
            IPO date: 2015
          </span>
        </CardMeta>
        <CardDescription>
          Week of 2018-03-29: 
          <br />
          open: 90.6100
          <br />
          high: 95.1390
          <br />
          low: 88.4000
          <br />
          close: 91.2700
          <br />
          adjusted close: 90.8771
          <br />
          volume: 207104405
          <br />
          dividend amount: 0.0000
        </CardDescription>
      </CardContent>
      <CardContent
        extra={true}
      >
        <a>
          <Icon
            as="i"
            name="chart line"
          />
          Bull
        </a>
      </CardContent>
    </Card>
  </Transition>
</div>
`;
